chart/templates/mongodb-service.yaml

```diff
@@ -8,9 +8,9 @@ metadata:
 spec:
   type: ClusterIP
   ports:
-  - name: mongodb
-    port: 27017
-    targetPort: mongodb
+  - protocol: TCP
+    port: 3940
```

Comment:
Can we leave the default mongodb port here?

Reply:
I had the same thought but I'm confused because of the downside of having the same value for `port` and `targetPort` - it might be unclear which of those two is actually the reason we have to use `27017` in the `url` const here:

https://github.com/okteto/movies/blob/main/api/server.js#L6

I am mentioning in the article though that `3940` is an arbitrarily chosen value so that should clear out any confusion folks might have. WDYT?

Reply:
@rberrelleza can I merge this? Need it for today's Services article.